cross/mosh/PLIST

```diff
@@ -0,0 +1,6 @@
+bin:bin/mosh
+bin:bin/mosh-client
+bin:bin/mosh-server
+rsc:share/man/man1/mosh-client.1
```

Comment:
We usually not include man pages in SynoCommunity packages

Reply:
Ok, I'll remove them then.